app/brave_main_delegate.cc

```diff
@@ -172,6 +173,9 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       features::kSmsReceiver.name,
       features::kWebXr.name,
       features::kWebXrGamepadModule.name,
+#if defined(OS_ANDROID)
+      feed::kInterestFeedContentSuggestions.name,
```

Comment:
https://github.com/brave/brave-browser/issues/7668
Follow-up issue created for considering re-enable this feature after we could support both NTP mode.